### PR TITLE
Fixes an error that occurred when a null value was set in the enum list

### PR DESCRIPTION
**Pull request by stanimoto**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-4.4.1
+14
```
Comment (osahan) [inline .nvmrc:1]: we are at https://github.com/themaven-net/tempest-schema/blob/master/.nvmrc 14.19.3 in tempest-schema

File: lib/bundler.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ function bundle(originPointer, originSchema, externalSchemaLoader) {
         else {
             for (var property in schemaPart) {
                 if (schemaPart.hasOwnProperty(property)) {
-                    if (typeof schemaPart[property] === 'object') {
+                    if (typeof schemaPart[property] === 'object' && schemaPart[property] !== null) {
```
Comment (pradheap) [inline lib/bundler.js]: nit: all conditions can be combined into one `if`?

**Review by osahan — CHANGES_REQUESTED** [review]

**Review by pradheap — APPROVED** [review]
LGTM except for one nit comment.

File: lib/bundler.js (hunk 2 above)
Comment (osahan) [reply lib/bundler.js]: I second that

File: .nvmrc (hunk 1 above)
Comment (stanimoto) [reply .nvmrc:1]: The intent here was to match the major version of what's referenced there at least, but I did not want to use a specific version.

**Pull request merged**